tensorflow/python/layers/base.py

```diff
@@ -354,3 +355,13 @@ def _add_elements_to_collection(elements, collection_list):
       if element not in collection_set:
         collection.append(element)
 
+def _should_add_regularizer(variable, existing_variable_set):
```

Comment:
Consider defining this utility in `add_weight` or in-lining its logic there, since it is only used in one place and it is a bit hard to understand out of context.